src/functional/fencil_processors/runners/gtfn_cpu.py

```diff
@@ -46,11 +61,21 @@ def convert_arg(arg):
     return [convert_arg(arg) for arg in args]
 
 
+def neighbortable_args(offset_provider):
+    return [c.tbl for c in offset_provider.values() if isinstance(c, Connectivity)]
```

Comment:
We need to define the Connectivity Protocol properly. `tbl` shouldn't be the API.